src/devices/onesti.ts

```diff
@@ -9,6 +9,24 @@ const e = exposes.presets;
 const ea = exposes.access;
 
 const fzLocal = {
+    nimly_lock_commandPinCodeRsp: {
```

Comment:
This converter is generic and can be moved to `fromZigbee.ts`

Reply:
Actually i think Nimly report 0 when successful in oppose of standard 1, i might have read wrong in the standard but...
i will chnge to lower leading letter

Reply:
i was wrong it is general
![image](https://github.com/user-attachments/assets/7f6afb45-47b9-4fd2-a2f4-fa9fafc29066)